docs/ref/settings.txt

```diff
@@ -3668,36 +3668,43 @@ Forms
 
 Globalization (``i18n``/``l10n``)
 ---------------------------------
-* :setting:`DATE_FORMAT`
-* :setting:`DATE_INPUT_FORMATS`
-* :setting:`DATETIME_FORMAT`
-* :setting:`DATETIME_INPUT_FORMATS`
-* :setting:`DECIMAL_SEPARATOR`
-* :setting:`FIRST_DAY_OF_WEEK`
-* :setting:`FORMAT_MODULE_PATH`
-* :setting:`LANGUAGE_CODE`
-* :setting:`LANGUAGE_COOKIE_AGE`
-* :setting:`LANGUAGE_COOKIE_DOMAIN`
-* :setting:`LANGUAGE_COOKIE_HTTPONLY`
-* :setting:`LANGUAGE_COOKIE_NAME`
-* :setting:`LANGUAGE_COOKIE_PATH`
-* :setting:`LANGUAGE_COOKIE_SAMESITE`
-* :setting:`LANGUAGE_COOKIE_SECURE`
-* :setting:`LANGUAGES`
-* :setting:`LANGUAGES_BIDI`
-* :setting:`LOCALE_PATHS`
-* :setting:`MONTH_DAY_FORMAT`
-* :setting:`NUMBER_GROUPING`
-* :setting:`SHORT_DATE_FORMAT`
-* :setting:`SHORT_DATETIME_FORMAT`
-* :setting:`THOUSAND_SEPARATOR`
-* :setting:`TIME_FORMAT`
-* :setting:`TIME_INPUT_FORMATS`
-* :setting:`TIME_ZONE`
-* :setting:`USE_I18N`
-* :setting:`USE_THOUSAND_SEPARATOR`
-* :setting:`USE_TZ`
-* :setting:`YEAR_MONTH_FORMAT`
+* i18n
```

Comment:
```suggestion

Internationalization (``i18n``)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
I think it might be nice to have these as sub-headers and remove the indent in the settings